README.md

```diff
@@ -116,30 +118,77 @@ terraform {
 ```
 
 ## Logging
-You can set the `TF_LOG` and `TF_LOG_PATH` environment variables to enable logging for Terraform. See the [official documentation](https://www.terraform.io/docs/internals/debugging.html) for details.
+You can set the `TF_LOG` and `TF_LOG_PATH` environment variables to enable 
+logging for Terraform. 
+See the [official documentation](https://www.terraform.io/docs/internals/debugging.html) 

```

Comment:
Looks like Hashicorp changed the URL to `https://www.terraform.io/internals/debugging`

Reply:
Fixed the link. Thanks.